### PR TITLE
support of telnet communication with NAD receivers

### PR DESCRIPTION
**Pull request by rymsha**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#19704
## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/media_player.nad.markdown [hunk 1]
```diff
@@ -27,8 +27,11 @@ media_player:
 {% configuration %}
 serial_port:
   description: The serial port.
-  required: true
-  default: "/dev/ttyUSB0"
+  required: false
```
Comment (frenck) [inline source/_components/media_player.nad.markdown:35]:
```suggestion
  required: exclusive
```

File: source/_components/media_player.nad.markdown [hunk 2]
```diff
@@ -27,8 +27,11 @@ media_player:
...
+  type: string
+host:
+  description: The IP address of your amplifier.
+  required: false
```
Comment (frenck) [inline source/_components/media_player.nad.markdown:40]:
```suggestion
  required: exclusive
```

File: source/_components/media_player.nad.markdown [hunk 3]
```diff
@@ -51,6 +54,8 @@ sources:
   type: [list, string]
 {% endconfiguration %}
 
+Either serial_port or host must be provided. If serial_port is provided RS-232 connection is established. If host is provided Telnet communication is established.
```
Comment (frenck) [inline source/_components/media_player.nad.markdown]:
```suggestion
Either `serial_port` or `host` must be provided. If a `serial_port` is provided, an RS-232 connection is established. If a `host` is provided, Telnet communication is established.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @rymsha!
I've reviewed it and provided some suggestions. Could you please take a look? Thanks! 👍

**Comment by rymsha** [comment]
Looks better. Thanks.

**Review by balloob — APPROVED** [review]

**Pull request merged**